### ISSUE TITLE
sveltekit: AuthApiError is not defined & Property 'status' does not exist on type 'AuthError'

### ISSUE THREAD

**Issue by cananda** [supabase/auth-helpers#378]
# Bug report

## Describe the bug

Discussed in https://github.com/supabase/supabase/discussions/10109

Originally posted by cananda November 5, 2022
I'm using all latest versions of sveltekit (535) and supabase-auth-helpers (0.8.6). Following the example here: https://github.com/supabase/auth-helpers/blob/main/packages/sveltekit/README.md

In this piece of code -
```
    if (error) {
      if (error instanceof AuthApiError && error.status === 400) {
        return invalid(400, {
          error: 'Invalid credentials.',
          values: {
            email
          }
        });
      }
```

.. I get AuthApiError is not defined and I also see 'Property 'status' does not exist on type 'AuthError'.ts(2339)' for error.status

Am I missing some imports?

## To Reproduce

Follow sveltekit example with latest sveltekit and auth-helper-sveltekit

- OS: Linux Fedora 37
- Browser (if applies): Chrome/Firefox
- Version of supabase-js: 2.0.6
- Version of Node.js: 18.7.0

**Comment by silentworks** [comment]
The example is missing the part where it imports `AuthApiError`, you will need to import that into the file first. I'll get the README updated and the docs updated too.

**Comment by silentworks** [comment]
Fixed in commit f1456ca
